icu4c/source/i18n/unicode/rbnf.h

```diff
@@ -90,18 +90,18 @@ enum URBNFRuleSetTag {
 /**
  * The RuleBasedNumberFormat class formats numbers according to a set of rules.
  *
- * <p>This number formatter is typically used for spelling out numeric values in words (e.g., 25,3476
+ * <p>This number formatter is typically used for spelling out numeric values in words (e.g., 25,376
  * as &quot;twenty-five thousand three hundred seventy-six&quot; or &quot;vingt-cinq mille trois
- * cents soixante-seize&quot; or
+ * cent soixante-seize&quot; or
  * &quot;f&uuml;nfundzwanzigtausenddreihundertsechsundsiebzig&quot;), but can also be used for
- * other complicated formatting tasks, such as formatting a number of seconds as hours,
- * minutes and seconds (e.g., 3,730 as &quot;1:02:10&quot;).</p>
+ * other complicated formatting tasks. For example, formatting a number as Roman numerals (e.g. 8 as VIII)
+ * or as ordinal digits (e.g. 1st, 2nd, 3rd, 4th).</p>
```

Comment:
Let's not advertise the deprecated duration rules, and mention the unmentioned numbering system rules.